ClaudiaRojasSoto/portfolio_1
# Form section mobile portfolio

Add a contact form and complete the mobile version of the portfolio website, as per the requirements of the third milestone.

Details:

- Add a contact form using the reference template.
- Implement HTML5 validation following these requirements:

1. Accept up to 30 characters in the name field.
2. Only accept emails in the email field.
3. Accept up to 500 characters in the message field.
4. Add placeholder text to all fields.
5. Ensure that all fields have the correct type.
6. Do not allow the user to submit the form if one or more fields are empty.

- Use the Formspree service to collect form data.
- Use Flexbox to layout the elements on the page.
- Make the website responsive using the following breakpoints:

1. The minimum mobile screen size should be 375px.
2. The minimum desktop screen size should be 768px.

- Implement button interactions.
- Follow best practices for HTML and CSS.
- Ensure that there are no linter errors.
- Document the work professionally and follow the correct GitHub flow.
Thank you for reviewing this pull request.
